lib/src/note/frequency.dart

```diff
@@ -0,0 +1,23 @@
+part of '../../music_notes.dart';
+
+/// Represents an absolute pitch, a physical frequency.
+class Frequency {
+  Frequency(this.hertz);
+
+  final double hertz;
+
+  /// Whether this [Frequency] is inside the human hearing range.
+  ///
+  /// Example:
+  /// ```dart
+  /// Note.a.inOctave(4).isHumanAudibleAt() == true
+  /// Note.d.inOctave(0).isHumanAudibleAt() == false
+  /// Note.g.inOctave(12).isHumanAudibleAt(442) == false
```

Comment:
Examples may be updated like:

```suggestion
  /// const Frequency(880).isHumanAudible == true
  /// Note.a.inOctave(4).equalTemperamentFrequency().isHumanAudible == true
  /// Note.g.inOctave(12).equalTemperamentFrequency(442).isHumanAudible == false
```